libs/traits/src/liquidity_pools.rs

```diff
@@ -37,6 +39,7 @@ pub trait LPEncoding: Sized {
 }
 
 /// The trait required for sending outbound messages.
+/// TODO: remove this
```

Comment:
Do we really want to remove the `Router` trait or is this comment outdated?

Reply:
We need it 👍🏻 along with the previous axelar precompile and routers

Reply:
We need it to remove I wanted to say 😆